lib/res/formatters/rspec.rb

```diff
@@ -65,11 +65,11 @@ def example_pending(pending_example_notification)
         set_status(result, test.metadata[:location], status)
       end
 
-        def add_result(test)
-       return { 
-          "type": "Rspec::Test",
-          "name": test[:description],
-          "urn": test[:location],
+      def add_result(test)
+        {
+          'type' => 'Rspec::Test',
```

Comment:
A better fix would be to just remove the quotes around the keys. Eg,:

```ruby
  type: "Rspec::Test"
```

as this will keep the keys as symbols rather than strings. This will then work for older Ruby versions.